iphone/Classes/TiUIPicker.m

```diff
@@ -190,19 +191,6 @@ -(void)setType_:(id)type_
 	}
 }
 
-// We're order-dependent on type being set first, so we need to make sure that anything that relies
-// on whether or not this is a date picker needs to be set AFTER the initial configuration.
--(void)setSelectionIndicator_:(id)value
```

Comment:
It was not used since iOS 7 anyway, so I decided to clean up the core directly. Having it or not does not have an effect to the developer.